### PR TITLE
Update to ruby-:two::one::three:

### PR DESCRIPTION
**Pull request by yeuem1vannam**
Refs: https://github.com/framgia/publify/issues/10
Update to ruby-:two::one::three:

``` sh
$ rspec spec/
# Finished in 2 minutes 12.3 seconds
#1395 examples, 0 failures, 3 pending
```

### CODE REVIEW

**Comment by yeuem1vannam** [comment]
@johnny-miyake @kouyaf77 Please review this PR

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ if ENV["HEROKU"]
   gem "thin" # Change this to another web server if you want (ie. unicorn, passenger, puma...)
   gem "rails_12factor"
 else
+  ruby "2.1.3"
```
Comment (j-miyake) [inline Gemfile:10]: Let's use 2.1.4! We should be on the edge :)

Comment (yeuem1vannam) [inline Gemfile:10]: Currently, we cannot install `2.1.4` via `rvm`, so if we use `2.1.4` it will be a hard for other developers, I think.

Comment (j-miyake) [inline Gemfile:10]: oh, I see.

**Comment by j-miyake** [comment]
OK

**Comment by oVuXuanDung** [comment]
ok

**Pull request merged**